src/test/data/JsonExpenseListStorageTest/invalidExpenseExpenseList.json

```diff
@@ -0,0 +1,8 @@
+{
+  "expenses": [ {
+    "name": "Expense with invalid name field: @!asda24",
```

Comment:
Is @ ! and numbers not permissible in the name?

Reply:
it says names only allow alphanumeric characters and spaces, so i dont think special characters are allowed?

Reply:
should we allow special char?